pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: dart_coveralls
-version: 0.4.0
+version: 0.5.0
 author: Axel Christ <adracus@gmail.com>
 description: |-
   Pub package to calculate coverage, format it
   to LCOV and send it to coveralls
 homepage: https://github.com/duse-io/dart-coveralls
 environment:
-  sdk: '>=1.9.0 <2.0.0'
+  sdk: '>=1.9.0 <1.22.0'
```

Comment:
I've not been in the dartland for a long time - It seems the dev branch is currently failing when this is restricted to 1.22 - Should this be adjusted or what is the reason for this?